pyproject.toml

```diff
@@ -4,15 +4,15 @@
 
 [tool.poetry]
 name = "ha-wyzeapi"
-version = "0.1.23"
+version = "0.1.24"
 description = "A Home Assistant integration for Wyze devices"
 authors = ["Katie Mulliken <katie@mulliken.net>"]
 license = "Apache-2.0"
 
 [tool.poetry.dependencies]
-python = "^3.10"
-homeassistant = "^2023"
-wyzeapy = "0.5.20"
+python = ">=3.11.0,<3.13"
```

Comment:
I'm a little out of the loop lately...
Looks good to me, but does this python range match home assistant?

Reply:
Currently, homeassistant needs version 3.11.0 or above. However, it also depends on ulid-transform (0.9.0) which needs version 3.10.0 to 4.0, and home-assistant-bluetooth (1.12.0) or (1.11.0) which both need version 3.11.0 to 3.13. It may seem a bit messy, but it was the only way to resolve the dependencies.